test/index.js

```diff
@@ -5,6 +5,7 @@
 var assert = require('assert');
 var vm = require('vm');
 
+var jsc = require('jsverify');
```

Comment:
I found the [answer][1]:

> ### Why do all the examples import the library as jsc instead of jsv?
>
> Does JSC originate with [JSCheck][2]?
>
> __A:__ Yes


[1]: https://github.com/jsverify/jsverify#why-do-all-the-examples-import-the-library-as-jsc-instead-of-jsv
[2]: http://www.jscheck.org/